app/templates/main/record.html

```diff
@@ -11,83 +11,84 @@
 {% block pageTitle %}Record – {{ config['SERVICE_NAME'] }} – GOV.UK{% endblock %}
 {% block content %}
     <div class="govuk-width-container govuk-width-container--record">
-        <main class="govuk-main-wrapper">
-            <div class="govuk-grid-row">
-                <div class="govuk-grid-row">
+        <div class="record-details">
+            <div class="search-record-page-desktop-view">{% include "top-search.html" %}</div>
+            <div class="govuk-grid-row govuk-grid-row--record-view">
+                <div class="govuk-grid-row govuk-grid-row-record">
                     <div class="govuk-grid-column-full govuk-grid-column-full__page-nav">
                         <p class="govuk-body-m govuk-body-m__record-view">You are viewing</p>
                         {% with dict = breadcrumbs %}
                             {% include "breadcrumb.html" %}
                         {% endwith %}
                     </div>
                 </div>
-                <div class="govuk-grid-column-two-thirds govuk-grid-column-two-thirds--record-search">
+                <div class="govuk-grid-column-two-thirds govuk-grid-column-two-thirds--record-table">
                     <h1 class="govuk-heading-m govuk-heading-m__record-details">Record details</h1>
-                    <dl class="govuk-summary-list govuk-summary-list-record--record">
+                    <dl class="govuk-summary-list govuk-summary-list--record">
                         <div class="govuk-summary-list__row"></div>
-                        <div class="govuk-summary-list__row govuk-summary-list-record__row--record">
-                            <dt class="govuk-summary-list__key govuk-summary-list-record__key--record-table">Filename</dt>
-                            <dd class="govuk-summary-list__value govuk-summary-list-record__value--record">
+                        <div class="govuk-summary-list__row govuk-summary-list__row--record">
+                            <dt class="govuk-summary-list__key govuk-summary-list__key--record-table">Filename</dt>
```

Comment:
are all these other style changes just tidy up of the record page since you're here?

Reply:
Correct @anthonyhashemi